packages/planner/src/app/components_ngrx/iterations-modal/iterations-modal.component.html

```diff
@@ -28,76 +17,59 @@
-            #startmydp="mydatepicker"
-            [options]="startDatePickerOptions"
-            (dateChanged)="onStartDateChanged($event)"
-            (calendarToggle)="onStartCalendarToggle($event)"
-            [(ngModel)]="startDate"></my-date-picker>
+          <my-date-picker #startmydp="mydatepicker" [options]="startDatePickerOptions" (dateChanged)="onStartDateChanged($event)"
+            (calendarToggle)="onStartCalendarToggle($event)" [(ngModel)]="startDate"></my-date-picker>
         </div>
         <div class="date-container padding-left-5 dib">
           <label>End Date</label>
-          <my-date-picker
-            #endmydp="mydatepicker"
-            [options]="endDatePickerOptions"
-            (dateChanged)="onEndDateChanged($event)"
-            (calendarToggle)="onEndCalendarToggle($event)"
-            [(ngModel)]="endDate"></my-date-picker>
+          <my-date-picker #endmydp="mydatepicker" [options]="endDatePickerOptions" (dateChanged)="onEndDateChanged($event)"
+            (calendarToggle)="onEndCalendarToggle($event)" [(ngModel)]="endDate"></my-date-picker>
         </div>
       </div>
       <div *ngIf="modalType=='create' || modalType=='update' || modalType=='createChild'">
         <label class="f8-active-label">Force Active: </label>
-          <switch (onChecked)="onChecked($event)"
-            [isChecked]="iteration.userActive" id="active-switch"></switch>
+        <switch (onChecked)="onChecked($event)" [isChecked]="iteration.userActive" id="active-switch"></switch>
       </div>
-      <div *ngIf="modalType=='close'"
-        class="form-group datepicker-container">
+      <div *ngIf="modalType=='close'" class="form-group datepicker-container">
         Do you really want to close this iteration?
       </div>
       <div class="error-block" [class.space-error]="spaceError">
-        <span class="help-block">There is already a current iteration. You must close the current iteration before starting this one.</span>
+        <span class="help-block">There is already a current iteration. You must close the current iteration before
+          starting this one.</span>
       </div>
     </div>
   </modal-content>
   <modal-footer>
     <div class="text-right col-xs-12 col-sm-12 col-md-12">
-      <button
-        [disabled]="submitLoading"
-        class="btn btn-default margin-right-10"
-        id="cancel-iteration-button" type="submit"
+      <button [disabled]="submitLoading" class="btn btn-default margin-right-10" id="cancel-iteration-button" type="submit"
         value="0" (click)="createUpdateIterationDialog.close()">
         Cancel
       </button>
-      <button class="btn btn-primary"
-        [disabled]="submitLoading"
-        id="create-iteration-button" type="submit" value="0"
-        (click)="createUpdateIterationDialog.onSubmit.emit(undefined)">
+      <button class="btn btn-primary" [disabled]="isNameFieldEmpty ||submitLoading" id="create-iteration-button" type="submit"
```

Comment:
@karthikjeeyar there should be a space between `||` and `submitLoading` and anyways the html formatting looks different from the way it should be.